<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(22, 71, 115);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>10</y>
     <width>751</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 190, 201);</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>INGRESAR REPUESTO</string>
    </attribute>
    <widget class="QLineEdit" name="txtCodigo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
border: 1px solid;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>CODIGO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnImagen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="text">
      <string>SELECCIONAR IMAGEN</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCategoria">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>SELECCIONAR LA CATEGORIA</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMoto">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>SELECCIONAR MOTO</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tRepuestos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>721</width>
       <height>192</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsert">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="text">
      <string>AGREGAR REPUESTO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>500</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="text">
      <string>SALIR</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtDescription">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>351</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid;
border-radius: 6px;</string>
     </property>
     <property name="placeholderText">
      <string>DESCRIPCION</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>INGRESAR LISTADO DE REPUESTOS</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>INGRESAR MOTO</string>
    </attribute>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
border: 1px solid;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>MOTO</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbYear">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>Modelo</string>
     </property>
     <item>
      <property name="text">
       <string>2000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2001</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2002</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2003</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2004</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2005</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2006</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2007</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2008</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2009</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2010</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2011</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2012</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2013</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2014</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2015</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2016</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnImoto">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="text">
      <string>AGREGAR MOTO</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tMoto">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>721</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>490</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(32, 32, 34);</string>
     </property>
     <property name="text">
      <string>SALIR</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtDmoto">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>391</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
border: 1px solid;</string>
     </property>
     <property name="placeholderText">
      <string>DESCRIPCION</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMarca">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>FiraCode Nerd Font</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
border: 1px solid;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>MARCA</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>ACTUALIZAR O ELIMINAR REPUESTO</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
